tenable/splunk-app-pvs
# Nnm updates

I've updated this app with:

- New branding/Naming
- A ton of filename changes
- Updates to all field names
- New sourectypes and parsing for NNM
- Tested support for prior sourcetypes
- Tested support for old (pvs) logs that are still coming in
- Standardized event naming 
- Simplified setup for customers. 
- Updated documentation
- Update root folder structure for easier building
- Improved field parsing regex for poorly formatted pvs/nnm syslog data 
- Remove a bunch of unused css
- Ensure app meets all pre-certification tests.
